packages/docs/docs/dev/connectors.mdx

```diff
@@ -3,78 +3,75 @@ title: Wallet Connectors
 category: For Developers
 ---
 
-# Wallet Connectors
+# Fuel Wallet Connectors
 
-A Wallet Connector is an interface to integrate with third-party wallets on the Fuel Ecosystem. It exposes all currently installed wallets in a unified manner
-via `window.fuel`.
+Fuel Wallet Connectors are a interface provided by Wallet developers to allow, you DApp to integrate with specific Wallets.
 
-<Examples.Connectors />
+You can learn more about how Fuel Wallet Connectors work, by reading the [Fuel Wallet Connectors](https://github.com/FuelLabs/fuels-wallet/wiki/Fuel-Wallet-Connectors) spec.
 
-## Methods
+## Setup Fuel Wallet Connectors
 
-### Listing connectors
-
-To list all available connectors you can use the `listConnectors()` method.
+The Fuel Wallet SDK allows you to provide a set of connectors, when creating a new instance.
+By default if you don't provide any connectors, the SDK will use the default connectors, `FuelWalletConnector` and `FueletWalletConnector`.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="listConnectors"
+  file="../../examples/connectors/ListConnectors.tsx"
+  commentBlock="createInstance"
 />
 
-### Selecting connector
+## Listing Connectors
+
+When working with mutiple connectors you should allow users to select the Connectors he want to use for interacting with your DApp.
+Once the `connectors()` method is called the Fuel Wallet SDK, will query informations from the connectors to allow you to know,
+waht connectors are installed.
 
-To select a different Wallet Connector use the `selectConnector(connectorName: string)` method.
+We also recommend to use the `connectors` listener on places that will use the `connectors` method as the availability can change.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="selectConnector"
+  file="../../examples/connectors/ListConnectors.tsx"
+  commentBlock="listConnectors"
 />
+<Examples.ListConnectors />
 
-## Events
+## Selecting Connector
 
-### CurrentConnector
-
-This event is triggered when the Current Connector is updated. The callback function receives the `FuelWalletConnector` object.
+Once you have a list of connectors, you can allow the user to select the connector he want to use. By using the methoid `selectConnect`.
+If the connector is not installed, the SDK will return false.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventCurrentConnector"
+  file="../../examples/connectors/SelectConnector.tsx"
```

Comment:
<img width="1512" alt="Screenshot 2023-11-23 at 10 46 06 AM" src="https://github.com/FuelLabs/fuels-wallet/assets/106365423/27c13f8e-a4d1-4fac-8fcc-360935c17dc0">
Not detecting Fuelet wallet or the non-development version of Fuel wallet

Reply:
They will need to update the wallet to work with the new connectors, we require a new method ping and version.